upgrade/sql/8.1.6-catchup.sql

```diff
@@ -0,0 +1,21 @@
+/* script intended for catching up with requests forgotten since 1.7 */
```

Comment:
Is the version 8.1.6 to be released? otherwise it might be better to rename it as 8.1.5-catchup.sql

Reply:
https://github.com/PrestaShop/PrestaShop/issues/35992